includes/sanitizers/class-amp-playbuzz-sanitizer.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+require_once( AMP__DIR__ . '/includes/sanitizers/class-amp-base-sanitizer.php' );
+
+/**
+ * Converts Playbuzz embed to <amp-playbuzz>
+ */
+class AMP_Playbuzz_Sanitizer extends AMP_Base_Sanitizer{
+
+    public static $tag = 'div';
+    public static $pb_class = 'pb_feed';
+
+
+
```

Comment:
Can we please remove the extra linebreaks here and through the rest of the file?